src/config/config.php

```diff
@@ -28,8 +28,8 @@
     'paths' => array(
         'api/*' => array(
             'allowedOrigins' => array('*'),
-            'allowedHeaders' => array('*'),
-            'allowedMethods' => array('*'),
+            'allowedHeaders' => array(),
+            'allowedMethods' => array('POST', 'PUT', 'GET', 'DELETE'),
```

Comment:
Because the description was confusing. It said first "allowedMethods can be set to array('*')" but right after "allowed methods however have to be explicitly listed". I'm actually forgetting PATCH.

If you check the standard, the latter is the correct description:

https://www.w3.org/TR/2014/REC-cors-20140116/#access-control-allow-headers-response-header

https://github.com/whatwg/fetch/issues/251